lib/engine/game/g_18_ga.rb

```diff
@@ -83,8 +83,8 @@ def operating_round(round_num)
           Step::G18GA::SpecialToken,
           Step::G18GA::BuyCompany,
           Step::HomeToken,
-          Step::G18GA::SpecialTrack,
-          Step::G18GA::Track,
```

Comment:
The Track and SpecialTrack classes for 18GA were just implementing the correct timing that can now be achieved with `"when": "Track"`. 18MEX, 18MS, and 18TN also enforce the timing and so their custom Step classes need to be looked at as well.